extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -147,11 +148,26 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				}
 			}));
 
+		this._sqlVmPageBlobInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				text: constants.DATABASE_BACKUP_SQL_VM_PAGE_BLOB_INFO,
+				style: 'information',
+				width: WIZARD_INPUT_COMPONENT_WIDTH,
+				CSSStyles: { ...styles.BODY_CSS, 'display': 'none' },
+				links: [
+					{
+						text: constants.DATABASE_BACKUP_SQL_VM_PAGE_BLOB_URL_LABEL,
+						url: 'https://learn.microsoft.com/en-us/azure/dms/known-issues-azure-sql-migration-azure-data-studio#azure-sql-managed-instance-and-sql-server-on-azure-virtual-machine-known-issues-and-limitations'
```

Comment:
please remove the locale from the URI, 'en-us' it will allow the browser to redirect to the article that matches the user locale

Reply:
good call, done